cg/models/rnafusion/rnafusion.py

```diff
@@ -93,3 +100,29 @@ class RnafusionAnalysis(AnalysisModel):
     """
 
     sample_metrics: Dict[str, RnafusionQCMetrics]
+
+
+class RnafusionDeliverables(PipelineDeliverables):
```

Comment:
I would restructure this class so that:
1. The deliverables template is a constant, probably outside the class
2. The`case_id` and `case_path` are parameters of the class
3. The function `get_deliverables_for_case` assigns the output to inherited attribute `files` instead of returning it, so that you don't need to call this function to instantiate the class.

Reply:
removed. see above